### DISCUSSION TITLE
No Issue - Just a Thank You :)

### DISCUSSION THREAD

**Discussion by fordmustang5l** [dabapps/django-forms-dynamic#7] [General]
Just wanted to drop some positivity and say how much I appreciate this package and what it has allowed me to do.

This package + HTMX + crispy_forms is ::chefs kiss:: perfect. Forms are such a core part of almost every business and you've made some pretty happy business people.

If you need help with docs, or cookbooks or any have ideas about this library and ways you need help, please let me know. Thanks

-j

**Comment by j4mie** [comment]
That's great to hear, thank you! I've just turned on Discussions so I'll send this over there for posterity.
